openmeter/credit/creditdriver/types.go

```diff
@@ -0,0 +1,21 @@
+package httpdriver
+
+import creditdriver "github.com/openmeterio/openmeter/internal/credit/driver"
+
+// requests
+type (
+	ListGrantsHandlerRequest = creditdriver.ListGrantsHandlerRequest
+	VoidGrantHandlerRequest  = creditdriver.VoidGrantHandlerRequest
+)
+
+// responses
+type (
+	ListGrantsHandlerResponse = creditdriver.ListGrantsHandlerResponse
+	VoidGrantHandlerResponse  = creditdriver.VoidGrantHandlerResponse
+)
+
+// params
+type (
+	ListGrantsHandlerParams = creditdriver.ListGrantsHandlerParams
+	VoidGrantHandlerParams  = creditdriver.VoidGrantHandlerParams
+)
```

Comment:
This will go away once we remove package aliasing, so nit: I prefer keeping types related to each other close. Package by feature vs package by layer.